src/newcharacter.cpp

```diff
@@ -3298,7 +3298,9 @@ tab_direction set_description( avatar &you, const bool allow_reroll,
                 no_name_entered = you.name.empty();
             }
             you.set_base_age( rng( 16, 55 ) );
-            you.set_base_height( rng( 145, 200 ) );
+            // Height distribution data is taken from this paper:
+            // http://www.biostat.jhsph.edu/bstcourse/bio751/papers/bimodalHeight.pdf
+            you.set_base_height( round( you.male ? normal_roll( 176.0, 7.4 ) : normal_roll( 162.8, 7.0 ) ) );
```

Comment:
```suggestion
            you.set_base_height( clamp( round( you.male ? normal_roll( 176.0, 7.4 ) : normal_roll( 162.8, 7.0 ) ), 145, 200 ) );
```

We should probably ensure that this stays inside the bounds the player can set themselves. The probability to go beyond is low but not 0.

Reply:
Thank you

Reply:
https://github.com/CleverRaven/Cataclysm-DDA/pull/39385
> I limited age to 16-55 and i limited height to 145-200 cm. the second is just about 2 std deviations from average, the bottom number is 2 std deviations below female average height, and the top number is 2 std deviations above male average height.

I don't think it's actually a problem if it's a normal roll now.

Reply:
> #39385
> 
> > I limited age to 16-55 and i limited height to 145-200 cm. the second is just about 2 std deviations from average, the bottom number is 2 std deviations below female average height, and the top number is 2 std deviations above male average height.
> 
> I don't think it's actually a problem if it's a normal roll now (though I guess it can still get results you can't manually input, so it probably is).

I did a test and found that in N=100000 about 0.035% males and 0.75% females generated out of bounds 145..200
https://onlinegdb.com/soRAitKPc